### ISSUE TITLE
[Suggestion]

### ISSUE THREAD

**Issue by kylelynskey** [AnthonyOSX/GO-Bot#119]
Having a window on the app with everything that has been captured or looted in the current session would be a nice touch because sometimes its hard to read the fast scrolling chat

**Comment by TheHydride** [comment]
I think if the Requests and Looking lines were hidden, it would make things clearer.

**Comment by kylelynskey** [comment]
Yeah true I never thought of it that way. The information is being given which is awesome but it cant be read at the moment

**Comment by jodriscoll** [comment]
@TheHydride I've submitted hiding the Request/Looking lines already. If you want to see things in the current session, open the log file in the directory and have at it. It will only stay up to date from the point of open and back. If you have a tool like Sublime Text, it will ask to update the file, allowing you to update it in increments. 

This should be sufficient until Anthony gets around to reviewing the request.
